edit/harvesterhci.io.setting/vm-force-deletion-policy.vue

```diff
@@ -0,0 +1,83 @@
+<script>
+import CreateEditView from '@/mixins/create-edit-view';
+import LabeledInput from '@/components/form/LabeledInput';
+import RadioGroup from '@/components/form/RadioGroup';
+
+export default {
+  name: 'HarvesterVMForceDeletePolicy',
+
+  components: { LabeledInput, RadioGroup },
+
+  mixins: [CreateEditView],
+
+  data() {
+    let parseDefaultValue = {};
+
+    try {
+      parseDefaultValue = JSON.parse(this.value.value);
+    } catch (error) {
+      parseDefaultValue = JSON.parse(this.value.default);
+    }
+
+    return {
+      parseDefaultValue,
+      errors: []
+    };
+  },
+
+  created() {
+    this.update();
+  },
+
+  methods: {
+    update() {
+      const value = JSON.stringify(this.parseDefaultValue);
+
+      this.$set(this.value, 'value', value);
+    }
+  },
+
+  watch: {
+    value: {
+      handler(neu) {
+        const parseDefaultValue = JSON.parse(neu.value);
+
+        this.$set(this, 'parseDefaultValue', parseDefaultValue);
+      },
+      deep: true
+    }
+  }
+};
+</script>
+
+<template>
+  <div class="row" @input="update">
+    <div class="col span-12">
+      <RadioGroup
+        v-model="parseDefaultValue.enable"
+        class="mb-20"
+        name="model"
+        :options="[true,false]"
+        :labels="[t('generic.enabled'), t('generic.disabled')]"
+        @input="update"
+      />
+
+      <LabeledInput
+        v-if="parseDefaultValue.enable"
+        v-model.number="parseDefaultValue.period"
+        class="mb-20"
+        :mode="mode"
+        label="period"
```

Comment:
i18n, and upper case

Reply:
I can input the letter, seems `v-model.number` doesn't work

![image](https://user-images.githubusercontent.com/18737885/144020544-804c71ae-d6f3-4f1a-bb43-73cc1aa9ee23.png)